src/lms-ds-loader/tests/test_lms_ds_loader.py

```diff
@@ -0,0 +1,5 @@
+from lms_ds_loader import __version__
+
+
+def test_version():
+    assert __version__ == '0.1.0'

```

Comment:
Should we keep this kind of auto-generated tests?

Reply:
I haven't decided what to do with this yet. If I don't add any real tests here then I'll remove the file.

Reply:
Removed this file.